configure.ac

```diff
@@ -314,9 +314,10 @@ dnl include all subsystem m4 fragments now that the core autoconf functionality
 dnl has been setup.  No fragment should do anything except define
 dnl PAC_SUBCFG_{PREREQ,BODY} macros which will be expanded later as
 dnl appropriate
-# begin subsys includes
-m4_include([subsys_include.m4])
-# end subsys includes
+m4_include([src/subconfigure.m4])
+
+m4_define([PAC_SUBCFG_MODULE_LIST],
+m4_dquote(PAC_SRC_SUBCFG_MODULE_LIST))
```

Comment:
What's the purpose of these and the other macros?  I looked through the code, but I don't see why they exist.

Reply:
Maybe I'm missing something.  `m4_include` is the expansion.  It basically places the content of the `subconfigure.m4` macro inline within the top-level `configure.ac`.  So what is `PAC_SUBCFG_MODULE_LIST` adding besides that?

Reply:
No `m4_include` adds the macro definitions, then it has to have the string to expand which is supplied by the second macro. Each one of these is built by `m4_map`. At least this my understanding.